KIPs/kip-81.md

```diff
@@ -0,0 +1,122 @@
+- The governance system should be able to manage the situations in which a particular entity makes an arbitrary decision. This is because the entity may weaken the sustainability of the entire network. 
+- The act of voting and the gain of voting power is different. 
+
+Governance Council members can exercise the right to vote based on the staking amount. Currently, each GC member receives at least 1 voting right regardless of the number of KLAY they hold. Recognizing a tendency that the member with more KLAY makes a decision that benefits the Klaytn ecosystem in the long run, we are granting a higher voting power to members with more KLAY. Yet, to prevent a particular subject from making an arbitrary decision, we are placing a cap on the voting power one can receive. 
+Therefore, the GC member will receive 1 vote per a certain amount of staked KLAY (initial configuration: 500 million KLAY). The maximum number of votes a GC member can own is one less than the total number of members. In other words, [Maximum Voting Power =  Total number of GC members - 1]. For example, if there are 35 GC members, one can own a maximum of 34 voting power. The 1 KLAY: 1 vote structure prevents the tragedy of the commons and potential monopolies by limiting the maximum number of votes cast. 
+The foundation will provide `votingDelay` (initial configuration: 7 days)  of the notice period and ‘votingPeriod’ of the voting period (initial configuration: 7 days). The list of eligible voters is finalized at the start of the notice period. The qualified electors and voting power of those will be counted at the start of the voting period. The foundation or GC can freely propose and discuss on the forum. Once the discussion is held on the forum, the foundation will put the topic to a vote on the governance portal. 
+The newly implemented governance portal includes the following functions: 
+- Ability to propose and vote on a variety of opinions
+- Information about Governance Council members who are working together for the Klaytn ecosystem
+
+The Klaytn governance voting system will be conducted on smart contracts. The contract design is influenced by Compound Finance and OpenZeppelin. The accounts interacting with the contracts can have proposer, voter, or executor roles.
+A proposer submits and initiates governance proposals. The voting timeline starts with the submission of proposals to the voting contract. A proposal describes the changes to be made on the Klaytn blockchain.
+A proposal not only contains textual descriptions of the change, but it may also include  specifications of attached transactions, or actions. The actions are executed on behalf of a voting contract after the proposal passes. The actions extend the impact of proposals from simply gathering opinions to automatically implementing changes on the blockchain.
```

Comment:
```suggestion
A proposal not only contains textual descriptions of the change, but it may also include specifications of attached transactions, or actions. The actions are executed on behalf of a voting contract after the proposal passes. The actions extend the impact of proposals from simply gathering opinions to automatically implementing changes on the blockchain.
```